src/common/account.js

```diff
@@ -111,7 +111,7 @@ class Account {
      * @param {boolean} ignoreZeroBalance true by default to hide zero balances
      */
     print(ignoreZeroBalance = true) {
-        let result = _.cloneDeep(this.assets);
```

Comment:
After upgrading to `@polkadot/api 10.9.1`, the following error occurs:

```
TypeError: Cannot read properties of undefined (reading 'createTypeUnsafe')
    at decode (/Users/star/devlab/xcm-demo/node_modules/@polkadot/api/node_modules/@polkadot/types/cjs/generic/ChainProperties.js:39:30)
```

The reason is that `_.cloneDeep` cannot be used on the `Balance` type.

Reply:
okay, _.cloneDeep is not really needed here since the `this.assets` is not modified.